.github/workflows/release-docker-github.yml

```
@@ -57,8 +63,17 @@
 
       - name: Set Sentry environment in .env
         run: |
-          echo "SENTRY_ENVIRONMENT=production" >> .env
-          echo "Set SENTRY_ENVIRONMENT=production in .env file"
+          # Create .env if it doesn't exist
+          touch .env
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)